hw/bsp/apollo3_evb/src/arch/cortex_m4/gcc_startup_apollo3.s

```diff
@@ -1,325 +1,329 @@
-/*

```

Comment:
What was changed in this file?

Reply:
I made it a unix format, and I added a small section that initializes the stack pointer.  The ambiq boot rom messes up the initial stack pointer setup.